lib/sentry.dart

```diff
@@ -494,3 +503,445 @@ class User {
     };
   }
 }
+
+/// The context interfaces provide additional context data.
```

Comment:
nit: here and throughout the PR add one blank line after the first paragraph in each dartdoc (see: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph)